sdk/contentsafety/Azure.AI.ContentSafety/src/Customization/TextBlocklist.Serialization.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Text.Json;
+using Azure;
+using Azure.Core;
+
+namespace Azure.AI.ContentSafety
+{
+    public partial class TextBlocklist : IUtf8JsonSerializable
+    {
+        void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
+        {
+            writer.WriteStartObject();
+            writer.WritePropertyName("blocklistName"u8);
+            writer.WriteStringValue(BlocklistName);
+            if (_isDiscriptionChanged)
```

Comment:
A potential problem here is `TextBlocklist` cannot be used in the other send-data scenario like `PUT`. We of course could add an `option` to tell these two different types of serialization. But this needs `ModelSerializerOptions` stuff release first. Please let me know your preference.

Reply:
I believe we discussed that current Azure API guidelines would prevent needing to serialize model content to both PUT and PATCH operations, so this shouldn't be an issue.  If there are existing services that need this, they have been "grandfathered in" using old guidelines, and it is a non-goal to support their APIs with generated code -- i.e. they will need to write these models and their custom per-operation serialization routines by hand.

When the new model reader-writer interfaces ship, generated models will need to serialize to both "tranferrable" and "persistable" formats, but you don't need to handle that case today.